handlers/Home/components/Services/styles.css

```diff
@@ -0,0 +1,135 @@
+/** @define HomeServices; use strict */
+
+.HomeServices-list {
+  border-top: 1px solid white;
+  margin: 0;
+  padding: 0;
+}
+
+.HomeServices-list:after {
+  content:'';
+  display:block;
+  clear: both;
+}
+
+.HomeServices-item {
+  border-bottom: 1px solid white;
+  border-right: 1px solid white;
+  box-sizing: border-box;
+  display: inline-block;
+  float: left;
+  height: 25vw;
+  margin-bottom: 0px;
+  position: relative;
+  width: 33.3333%;
+  background-size: cover;
+}
+
+.HomeServices-item:after, .HomeServices-item:before {
```

Comment:
I swear I’m the only person in all of Skookum that refuses to do single-line comma selectors (here and a few down)